Трохимец Сергий/First HomeWork/Task2.rb

```diff
@@ -0,0 +1,5 @@
+# Task 2
+hash = {'Yes' => 23, 'b' => 'travel', 'yesterday' => 34, 5 => '234',
+ yesss: :fg, try: 30, key: 'some value', ' yesterday1 '=> 34,' yesteryear '=> 2014}
+ p hash.count{ |key, _| key =~ /^yes/}
```

Comment:
```suggestion
 p hash.count{ |key, _| key =~ /^yes/ }
```

https://github.com/rubocop-hq/ruby-style-guide#spaces-operators
It's better to add spaces so every part of your code is stand out visually